### PR TITLE
[8.0][website_mail_snippet_responsive] Pseudoresponsive mail snippets.

### PR DESCRIPTION
**Pull request by yajo**
Responsive is a hard thing in mails, because mail clients usually render very badly everything.

This new module adds a good collection of snippets in a pseudoresponsive manner, that will seem responsive in most cases, although they are not, technically.

Inspired by https://css-tricks.com/ideas-behind-responsive-emails/.

@rafaelbn @dgzurita

### CODE REVIEW

**Comment by dgzurita** [comment]
Tested in runbot :+1:

File: website_mail_snippet_responsive/README.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+=============================================
+Responsive Layout Snippets for Writing Emails
+=============================================
+
+This module extends the functionality of the website mail designer to support a
+responsive layout and allow you to read those mails comfortably in a mobile
+mail client.
+
+Well... actually this is not really responsive. It is pseudo-responsive.
+Responsiveness in current mail clients sucks a lot, so these templates are very
```
Comment (pedrobaeza) [inline website_mail_snippet_responsive/README.rst]: Avoid the word "sucks". It's not too polite.

File: website_mail_snippet_responsive/README.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Install any module that makes use of the website mail designer, such as
+   ``mass_mailing``.
+#. Edit an email with the website mail designer.
+#. You have a new collection of snippets. Use them as usual.
```
Comment (pedrobaeza) [inline website_mail_snippet_responsive/README.rst]: In which section you have these new snippets?

File: website_mail_snippet_responsive/README.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+line, and then use the option that will float above it.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/205/8.0
+
+Known issues / Roadmap
+======================
+
+* Snippet code sucks, but that's because mail client HTML engines suck. I
```
Comment (elicoidal) [inline website_mail_snippet_responsive/README.rst]: Please rephrase ;)

**Comment by elicoidal** [comment]
LGTM (no test)

**Comment by yajo** [comment]
Sorry, it does not sound so bad in my mother language. Fixed.

**Comment by rafaelbn** [comment]
Tested and used in production enviroment :+1:

**Comment by sergio-teruel** [comment]
@Yajo Really good. :+1:

**Comment by pedrobaeza** [comment]
One last question. Why not add a new section for your widgets, for differentiating from the existing ones?

**Comment by carlosdauden** [comment]
:+1:

**Comment by rafaelbn** [comment]
Hi @pedrobaeza this is the same way Odoo do. I quite clear. A section you mean "structure", "contect", feature" ... ? This snnipet are structure so I think it's a good place.

**Comment by pedrobaeza** [comment]
I mean they are in the section "Email design". Maybe it can be put in another section called "Email design (enhanced)" or similar for splitting standard ones from these.

**Comment by rafaelbn** [comment]
OK, I understand  what you say. IMHO I would like to have the best new "responsive" snippets in my first screen if not I will have to change of section each time. Furthermore, this snippets are Email design, they don't enhance anything they are just well done in comparison with standards ones.

**Comment by pedrobaeza** [comment]
OK :+1:

**Pull request merged**